### DISCUSSION TITLE
使用 ref 模板引用在组件生命周期内随时获取cesium和viewer变量

### DISCUSSION THREAD

**Discussion by CuriousAccordion** [zouyaoji/vue-cesium#647] [Q&A] [answered]
文档中说到
你也可以使用 ref 模板引用在组件生命周期内随时获取这两个变量。
有没有例子？如果没有的话，我想通过这种方式使用，该如何操作？

**Comment by zouyaoji** [comment, marked as answer]
https://zouyaoji.top/vue-cesium/#/zh-CN/component/quickstart
